avocado/plugins/spawners/lxc.py

```diff
@@ -105,6 +105,9 @@ class LXCSpawner(Spawner, SpawnerMixin):
     METHODS = [SpawnMethod.STANDALONE_EXECUTABLE]
     slots_cache = {}
 
+    def is_operational(self):
+        return True
```

Comment:
Since on the LXC spawner we also have some preliminary check we were doing while spawning tasks (can be moved to the `is_operational` method now) like
```python
        if not LXC_AVAILABLE:
            msg = "LXC python bindings not available on the system"
            runtime_task.status = msg
            return False
```
I would recommend we even take the above a tiny step further:
```diff
+    def is_operational(self):
+        msg = "LXC python bindings not available on the system"
+        LOG.error(msg)
+        return LXC_AVAILABLE
```
with the error message provided in the logs too. I guess we could then also remove the LXC_AVAILABLE check in the original `spawn_task()` method.

Reply:
Updated with your suggestion.